lib/DAV/WorkspacePlugin.php

```diff
@@ -98,9 +99,12 @@ public function propFind(PropFind $propFind, INode $node) {
 			$nodes = $this->rootFolder->getUserFolder($this->userId)->getById($node->getId());
 			if (count($nodes) > 0) {
 				/** @var File $file */
-				$file = $this->workspaceService->getFile($nodes[0]);
-				if ($file instanceof File) {
-					return $file->getContent();
+				try {
+					$file = $this->workspaceService->getFile($nodes[0]);
+					if ($file instanceof File) {
+						return $file->getContent();
+					}
+				} catch (StorageNotAvailableException $e) {
```

Comment:
mind adding a code comment about why the exception is ignored silently?

Reply:
sure